packages/gafl-webapp-service/src/pages/contact/digital-licence/licence-confirmation-method/result-function.js

```diff
@@ -1,6 +1,11 @@
-import { CONTACT_SUMMARY_SEEN, CommonResults } from '../../../../constants.js'
+import { CONTACT_SUMMARY_SEEN, CommonResults, CHANGE_CONTACT_DETAILS_SEEN } from '../../../../constants.js'
 
 export default async request => {
   const status = await request.cache().helpers.status.getCurrentPermission()
-  return status.fromSummary === CONTACT_SUMMARY_SEEN ? CommonResults.SUMMARY : CommonResults.OK
+
+  return status.fromContactDetailsSeen === CHANGE_CONTACT_DETAILS_SEEN.SEEN
+    ? CommonResults.AMEND
+    : status.fromSummary === CONTACT_SUMMARY_SEEN
+      ? CommonResults.SUMMARY
+      : CommonResults.OK
```

Comment:
Another really complex statement. I defy you to read and understand this quickly a few months from now, needs to refactor to an `if`